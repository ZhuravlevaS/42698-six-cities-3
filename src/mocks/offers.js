export default [
  {
    img: `img/apartment-01.jpg`,
    price: 120,
    rating: 4,
    description: `Beautiful & luxurious apartment at great location`,
    type: `Apartment`,
    isMarked: false,
    isPremium: true,
    id: 1
  },

  {
    img: `img/room.jpg`,
    price: 80,
    rating: 4,
    description: `Wood and stone place`,
    type: `Private room`,
    isMarked: true,
    isPremium: false,
    id: 2
  },

  {
    img: `img/apartment-02.jpg`,
    price: 132,
    rating: 4,
    description: `Canal View Prinsengracht`,
    type: `Apartment`,
    isMarked: false,
    isPremium: false,
    id: 3
  },

  {
    img: `img/apartment-03.jpg`,
    price: 180,
    rating: 5,
    description: `Nice, cozy, warm big bed apartment`,
    type: `Apartment`,
    isMarked: false,
    isPremium: false,
    id: 4
  },

  {
    img: `img/room.jpg`,
    price: 80,
    rating: 4,
    description: `Wood and stone place`,
    type: `Private room`,
    isMarked: false,
    isPremium: false,
    id: 5
  }
];
