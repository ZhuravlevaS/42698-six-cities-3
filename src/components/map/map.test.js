import React from 'react';
import Map from './map.jsx';
import renderer from 'react-test-renderer';
import offers from '../../mocks/offers.js';

const cityCords = [52.38333, 4.9];
const cords = offers.map((apart) => apart.cords);

it(`<Map/> render`, () => {
  const tree = renderer
    .create(<Map city={cityCords} cords={cords}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
