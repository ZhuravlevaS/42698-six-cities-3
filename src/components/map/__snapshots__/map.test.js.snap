// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Map/> render 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "calc(100vh - 185px)",
      "width": "100%",
    }
  }
/>
`;
